docs/tutorials/http-serving-tutorial.md

```diff
@@ -13,18 +13,17 @@ Here's an example of a new function with an endpoint decorator:
 
 ```javascript
 @GetApi('/greeting/:name')
-static async greetingEndpoint(ctx: HandlerContext, @ArgSource(ArgSources.URL) name: string) {
+static async greetingEndpoint(ctx: HandlerContext, name: string) {
   return `Greeting, ${name}`;
 }
 ```
 Here's an example applying an endpoint decorator to an existing transaction:
 
 ```javascript
-@PostApi('/clear/:user')
+@PostApi('/greeting/:friend')
 @Transaction()
-static async clearTransaction(ctxt: TransactionContext<Knex>, @ArgSource(ArgSources.URL) user: string) {
-  await ctxt.client.raw("DELETE FROM dbos_hello WHERE NAME = ?", [user]);
-  return `Cleared greet_count for ${user}!\n`;
+static async insertGreeting(ctxt: TransactionContext<Knex>, friend: string, note: string) {
+  await ctxt.client.raw('INSERT INTO greetings (name, note) VALUES (?, ?)', [friend, note]);
```

Comment:
We used `dbos_greetings` and `dbos_hello` everywhere so people can directly change their code and run `npx dbos start`. Now this has dependency to programming guide.